osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -150,7 +150,11 @@ public override bool OnExiting(IScreen next)
         protected void StartPlay()
         {
             sampleStart?.Play();
-            ParentScreen?.Push(CreateGameplayScreen());
+
+            // fallback is to allow this class to operate when there is no parent OnlineScreen (testing purposes).
+            var targetScreen = (Screen)ParentScreen ?? this;
```

Comment:
I've generally tried to avoid this to keep logic without these special cases. `TestSceneMultiplayer` can be used to test this, which already has examples.

But I don't think this is too bad... I think... So I'll go with it for now.